### PR TITLE
Updated docstrings for ramp_fit

### PR DESCRIPTION
**Pull request by dmggh**

### CODE REVIEW

File: jwst/ramp_fitting/ramp_fit.py [hunk 1]
```diff
@@ -1086,63 +1072,63 @@ def calc_slope(data_sect, gdq_sect, frame_time, opt_res, save_opt, rn_sect,
 
     Parameters
     ----------
-    data_sect: float
+    data_sect : float
         section of input data cube array
 
-    gdq_sect: float
+    gdq_sect : float
```
Comment (philhodge) [inline jwst/ramp_fitting/ramp_fit.py]: This should be an int array, not a float.

File: jwst/ramp_fitting/ramp_fit.py [hunk 2]
```diff
@@ -45,43 +43,43 @@ def ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
...
 
     Returns
     -------
-    new_model: Data Model object
+    new_model : Data Model object
         DM object containing a rate image averaged over all integrations in
         the exposure
 
-    int_model: Data Model object
+    int_model : Data Model object
```
[the remark this answers is not in the document]
Comment (hbushouse) [reply jwst/ramp_fitting/ramp_fit.py]: 👍

File: jwst/ramp_fitting/ramp_fit.py [hunk 3]
```diff
@@ -1009,15 +999,15 @@ def dq_compress_sect(gdq_sect, pixeldq_sect):
 
     Parameters
     ----------
-    gdq_sect: int (uint8), 3D array
+    gdq_sect : int (uint8), 3D array
         cube of GROUPDQ array for a data section
 
-    pixeldq_sect: int (uint16), 2D array
+    pixeldq_sect : int (uint16), 2D array
```
Comment (hbushouse) [inline jwst/ramp_fitting/ramp_fit.py]: Is the data type of this array still uint16? The pixeldq array is now up to uint32. Maybe just list it as "int" and don't worry about the specific size.

File: jwst/ramp_fitting/ramp_fit.py [hunk 4]
```diff
@@ -1009,15 +999,15 @@ def dq_compress_sect(gdq_sect, pixeldq_sect):
...
         dq array of data section of input model
 
     Returns
     -------
-    pixeldq_sect: int (uint16), 2D array
+    pixeldq_sect : int (uint16), 2D array
```
Comment (hbushouse) [inline jwst/ramp_fitting/ramp_fit.py]: uint16 or uint32?

**Pull request merged**